shared/applicability/bootc.yml

```diff
@@ -0,0 +1,18 @@
+name: cpe:/a:bootc
```

Comment:
This also applies to other immutable OSes like Fedora Silverblue.

Should we rename to something like immutable?